infor-design/enterprise-wc
# 1672 - IdsChartLegendMixin: Add 'none' setting, change usage in IdsPieChart

**Explain the details for making this change. What existing problem does the pull request solve?**
This PR changes the way the `legend-placement` attribute works in IdsChartLegendMixin, which solves a couple issues:
- Removes the need to default the legend placement to 'right', which fixes a problem when trying to set the attribute in the Angular examples
- Also naturally creates a default where legends aren't present in charts using the new `none` value.

**Related github/jira issue (required)**:
Related to #1672

**Steps necessary to review your pull request (required)**:
Pull/build/run, then smoke test the following chart example pages.  There should be no visual discrepancies:
- http://localhost:4300/ids-area-chart
- http://localhost:4300/ids-axis-chart
- http://localhost:4300/ids-bar-chart
- http://localhost:4300/ids-line-chart
- http://localhost:4300/ids-pie-chart

_Test new settings:_
- Open http://localhost:4300/ids-pie-chart/legend-placement.html
- By default, no legend should be present
- Try setting the different values in the radio group to change the visibility/layout of the legend

_Test in Angular examples:_
- Run `npm run publish:link` on this branch, and test with infor-design/enterprise-wc-examples#52

**Included in this Pull Request**:
